homeassistant/components/humidifier/__init__.py

```diff
@@ -34,8 +35,6 @@
     ATTR_MIN_HUMIDITY,
     DEFAULT_MAX_HUMIDITY,
     DEFAULT_MIN_HUMIDITY,
-    DEVICE_CLASS_DEHUMIDIFIER,
```

Comment:
We still need to import the names here to not break integrations that import from the root package.

Reply:
Oh right!